scss/topic-cards.scss

```diff
@@ -37,19 +38,33 @@
   grid-column-gap: 12px;
   grid-row-gap: 8px;
   border-radius: var(--d-border-radius);
+  td.main-link .link-top-line {
```

Comment:
out of curiosity, why is this not done with grid-area?

Reply:
Using grid area here would require rewriting grid area definitions for every break point & variation in the card. It would work, but would add a lot of text to the file. So i chose the less verbose route.